Dockerfile.mail

```diff
@@ -43,6 +43,7 @@ RUN adduser --system --uid 1001 nextjs
 
 # Copy built application
 COPY --from=build /app /app
+COPY --from=build /app/src/server/mail-service/ejsTemplates /app/dist/ejsTemplates
```

Comment:
Why not puting it in the public folder?

Reply:
It was done by omnistrate, I didn't want to change too much.